pkg/mirrorregistries/generator.go

```diff
@@ -59,7 +63,12 @@ func (m *mirrorRegistriesConfigBuilder) IsMirrorRegistriesConfigured() bool {
 // the mirror registries are not configured.
 // empty dir is due to the way we mao configmap in the assisted-service pod
 func (m *mirrorRegistriesConfigBuilder) GetMirrorCA() ([]byte, error) {
-	return os.ReadFile(m.MirrorRegistriesCertificatePath)
+	bytes, err := m.fileReader(m.MirrorRegistriesCertificatePath)
+	if err != nil {
+		// fallback to tls-ca-bundle.pem (used by ABI)
+		return m.fileReader(common.SystemCertificateBundlePath)
```

Comment:
Instead of injecting the reader function, can we set this path on the struct?
It will be less likely to confuse folks in the future that don't know why this `fileReader` exists

Reply:
Sure, good idea.